aeron-archive/src/main/java/io/aeron/archive/ReplicationSession.java

```diff
@@ -146,6 +146,8 @@ enum State
         {
             isDestinationRecordingEmpty = false;
         }
+
+        logStarted(replicationId, controlSession.sessionId(), srcRecordingId, dstRecordingId, replicationChannel);
```

Comment:
I do not think we need an additional `logStarted` method. The start of the replication is logged via `CMD_IN_REPLICATE` or `CMD_IN_REPLICATE2` events.